diagnostic.md

```diff
@@ -23,7 +27,7 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
-
+$('#li').find('#form');
```

Comment:
This will select an element with an `id` of `li` and then find all (there can only be one in this case) descendants of the element with an `id` of `form`.  A few possible ways to navigate from a `<li>` element to the `<form>` element are `$('li').parent().parent().find('form')`, `$('li').parent().parent().children('form')`, and `$('.listerItem').parent().parent().$('form')`.